### PR TITLE
nextcloud-client 2.3.2

### PR DESCRIPTION
**Pull request by caugner**
###### Motivation for this change

I'm switching from Fedora to NixOS and would like to continue using the Nextcloud themed version of the owncloud-client.

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x]  Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/networking/nextcloud-client/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  ];
+
+  preConfigure = ''
+    cmakeFlags="$cmakeFlags -DOEM_THEME_DIR=$(realpath ./nextcloudtheme) ../client"
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Nextcloud themed desktop client";
+    homepage = https://nextcloud.com;
+    maintainers = [ ];
```
Comment (Mic92) [inline pkgs/applications/networking/nextcloud-client/default.nix]: Did you indent to add your alias here?

Comment (caugner) [reply pkgs/applications/networking/nextcloud-client/default.nix]: Thank you, I did, and fixed that now.

Comment (caugner) [reply pkgs/applications/networking/nextcloud-client/default.nix]: Also, I have added the license now.

**Comment by Mic92** [comment]
Thanks!

**Comment by caugner** [comment]
Thanks @Mic92!

By the way, are you aware of a direct way to obtain the sha256 hash for the git repository in connection with `fetchSubmodules = true`? In my case I first put an arbitrary hash, tried to build (which obviously failed) and then copied the hash from the failure message. However, that felt a bit like a dirty hack…

**Pull request merged**